docs/resources/egress_private_endpoint.md

```diff
@@ -0,0 +1,173 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "cockroach_egress_private_endpoint Resource - terraform-provider-cockroach"
+subcategory: ""
+description: |-
+  Egress private endpoint for a CockroachDB Cloud cluster. Egress private endpoints allow CockroachDB Cloud clusters to connect to external services through private network connections without traversing the public internet.
+  - Terraform resource docs https://github.com/cockroachdb/terraform-provider-cockroach/blob/main/docs/resources/egress_private_endpoint.md
```

Comment:
Suggest renaming to "Terraform resource docs source" if we want to keep this link at all